lib/validate_encrypt_utils.pm

```diff
@@ -83,16 +99,17 @@ sub verify_cryptsetup_properties {
 sub verify_restoring_luks_backups {
     my (%args)           = @_;
     my $mapped_dev       = $args{encrypted_device};
+    my $mapped_dev_path  = $args{encrypted_device_path};
     my $backup_file_info = $args{backup_file_info};
-    my $backup_path      = "/root/$mapped_dev";
+    my $backup_path      = $args{backup_base_path} . '_' . $mapped_dev;
```

Comment:
So, as per out discussion, I would either have hardcoded string in test data, or some generated name, like you do here. Main point is that we don't need both `encrypted_device_path` and `encrypted_device` as now `encrypted_device` is only used to generate path for the backup header file. So tbh, I don't see a need to pass `encrypted_device`.

So I'm fine with both solutions, generate path dyncamically using device name, or use static file, but I would avoid mixture of both approaches.

Reply:
i worked around this. and `encrypted_device` is not needed anymore in the sub